dotcom-rendering/src/components/AusEoy2024Wrapper.importable.tsx

```diff
@@ -0,0 +1,299 @@
+import { ChoiceCards } from './marketing/banners/designableBanner/components/choiceCards/ChoiceCards';
+import { buttonStyles } from './marketing/banners/designableBanner/styles/buttonStyles';
+import { useChoiceCards } from './marketing/hooks/useChoiceCards';
+import type { ReactComponent } from './marketing/lib/ReactComponent';
+
+const styles = {
+	container: css`
+		/* stylelint-disable-next-line color-no-hex */
+		background: #041d35;
+		color: ${palette.neutral[100]};
+	`,
+	grid: css`
+		display: grid;
+		flex-direction: column;
+		position: relative;
+		padding: 0 ${space[3]}px ${space[4]}px ${space[3]}px;
+		${from.tablet} {
+			display: grid;
+			grid-template-columns: 350px 350px;
+			padding: 0 ${space[5]}px ${space[4]}px ${space[5]}px;
+		}
+		${from.desktop} {
+			grid-template-columns: 462px 462px;
+			column-gap: ${space[4]}px;
+		}
+		${from.leftCol} {
+			grid-template-columns: 148px 482px 482px;
+			column-gap: 0;
+		}
+		${from.wide} {
+			grid-template-columns: 228px 482px 482px;
+		}
+	`,
+	logo: css`
+		display: none;
+		${from.leftCol} {
+			display: block;
+			grid-column: 1;
+			grid-row: 1;
+			margin-top: ${space[2]}px;
+		}
+	`,
+	heading: css`
+		${from.tablet} {
+			grid-column: 1;
+			grid-row: 1;
+		}
+		h2 {
+			margin: ${space[2]}px 0 ${space[4]}px;
+			color: ${palette.neutral[100]};
+
+			${headlineMedium24}
+			${from.tablet} {
+				${headlineMedium28}
+			}
+			${from.leftCol} {
+				${headlineMedium34}
+			}
+		}
+		${from.leftCol} {
+			grid-column: 2;
+			border-left: 1px solid rgba(255, 255, 255, 0.2);
+			padding-left: ${space[2]}px;
+		}
+	`,
+	body: css`
+		${textEgyptian17};
+		strong {
+			${textEgyptianBold17};
+		}
+	`,
+	ticker: css`
+		margin-bottom: ${space[4]}px;
+	`,
+	choiceCards: css`
+		margin-top: ${space[6]}px;
+		${from.tablet} {
+			grid-column: 2;
+			grid-row: 1;
+			align-self: flex-start;
+			display: flex;
+			justify-content: flex-end;
+		}
+		${from.leftCol} {
+			grid-column: 3;
+			margin-right: ${space[3]}px;
+		}
+	`,
+};
+
+const tickerSettings = {
+	currencySymbol: '$',
+	copy: {},
+	tickerStylingSettings: {
+		filledProgressColour: '#FFBAC8',
+		progressBarBackgroundColour: '#1F4153',
+		headlineColour: '#000000',
+		totalColour: '#FEBBC7',
+		goalColour: '#FFFFFF',
+	},
+};
+
+const getTickerData = async (): Promise<TickerData | undefined> => {
+	const data = await fetch(
+		'https://support.theguardian.com/ticker/AU.json',
+	).then((response) => response.json());
+
+	if (isObject(data)) {
+		const { total, goal } = data;
+		if (typeof total === 'number' && typeof goal === 'number') {
+			return {
+				total: Math.floor(total),
+				goal: Math.floor(goal),
+			};
+		}
+	}
+	return;
+};
+
+const choiceCardAmounts: SelectedAmountsVariant = {
+	testName: 'au-eoy-front-amounts',
+	variantName: 'CONTROL',
+	defaultContributionType: 'MONTHLY',
+	displayContributionType: ['ONE_OFF', 'MONTHLY', 'ANNUAL'],
+	amountsCardData: {
```

Comment:
are these amounts taken from the US version? These should probably match the AU amounts in the RRCP amounts tool

Reply:
oh good point!!

Reply:
updated as per the RRCP amount tool